src/Sylius/Bundle/AdminBundle/Console/Command/ListAdminUsersCommand.php

```diff
@@ -0,0 +1,127 @@
+
+    /** @param UserRepositoryInterface<AdminUserInterface> $userRepository */
+    public function __construct(
+        private readonly UserRepositoryInterface $userRepository,
+    ) {
+        parent::__construct();
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function configure(): void
+    {
+        $this->addOption(
+            'search',
+            's',
+            InputOption::VALUE_REQUIRED,
+            'Search users by email, username, first name or last name (partial match)',
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->io->title('List available admin users');
+        $searchTerm = $input->getOption('search');
+        if ($searchTerm === null || $searchTerm === '') {
+            $this->renderUsersTable($output, $this->userRepository->findAll());
+
+            return Command::SUCCESS;
+        }
+        $users = $this->searchUsers($searchTerm);
+        if (empty($users)) {
+            $this->io->warning(sprintf('No users found matching "%s"', $searchTerm));
+
+            return Command::SUCCESS;
+        }
+        $this->io->success(sprintf('Found %d user(s) matching "%s"', count($users), $searchTerm));
+        $this->renderUsersTable($output, $users);
+
+        return Command::SUCCESS;
+    }
+
+    /**
+     * @return array<AdminUserInterface>
+     */
+    private function searchUsers(string $searchTerm): array
+    {
+        $searchLower = mb_strtolower($searchTerm);
+
+        return array_filter($this->userRepository->findAll(), function (AdminUserInterface $user) use ($searchLower): bool {
+            $email = mb_strtolower($user->getEmail() ?? '');
+            $username = mb_strtolower($user->getUsername() ?? '');
+            $firstName = mb_strtolower($user->getFirstName() ?? '');
+            $lastName = mb_strtolower($user->getLastName() ?? '');
+
+            return str_contains($email, $searchLower) ||
+                str_contains($username, $searchLower) ||
+                str_contains($firstName, $searchLower) ||
+                str_contains($lastName, $searchLower);
+        });
+    }
+
+    /**
+     * @param array<AdminUserInterface> $adminUsers
+     */
+    private function renderUsersTable(OutputInterface $output, array $adminUsers): void
+    {
+        $table = new Table($output);
+        $table->setHeaders([
+            'ID', 'E-Mail', 'Username', 'First name', 'Last name', 'Locale code', 'Enabled', 'Roles',
+        ]);
+
+        foreach ($adminUsers as $adminUser) {
+            $table->addRow([
+                $adminUser->getId(),
+                $adminUser->getEmail(),
+                $adminUser->getUsername(),
+                $adminUser->getFirstName() ?: 'No first name set',
+                $adminUser->getLastName() ?: 'No last name set',
```

Comment:
Empty value should be enough.

```suggestion
                $adminUser->getFirstName() ?? '',
                $adminUser->getLastName() ?? '',
```

Reply:
What speaks against saying: "No last/first name set"?